### DISCUSSION TITLE
Cypress testing targeting WebView2

### DISCUSSION THREAD

**Discussion by sorensenmatias** [cypress-io/cypress#15059] [Questions and Help]
Hi,

Is it possible to use cypress to test embedded webpage in WebView2 in my native Windows app?
Can I maybe somehow point cypress to the embedded Chromium and then test as normal from there?

For a description of WebView2 see here: https://docs.microsoft.com/en-us/microsoft-edge/webview2/

`The Microsoft Edge WebView2 control enables you to embed web technologies (HTML, CSS, and JavaScript) in your native applications. The WebView2 control uses Microsoft Edge (Chromium) as the rendering engine to display the web content in native applications. `

**Comment by bahmutov** [comment]
Unfortunately, Cypress can only control specific stand-alone browsers:
- built-in Electron browser
- Chrome 
- Edge
- Firefox
